src/bootstrap/src/core/builder/mod.rs

```diff
@@ -1069,6 +1069,11 @@ impl<'a> Builder<'a> {
                 doc::EditionGuide,
                 doc::StyleGuide,
                 doc::Tidy,
+                doc::Bootstrap,
+                doc::Releases,
+                doc::RunMakeSupport,
+                doc::BuildHelper,
+                doc::Compiletest,
```

Comment:
Note that doing this will likely result in more merge conflicts going forward, as upstream will add new docs at the bottom of the list. In principle I really appreciate this change, but it might be painful for PR rotation.

Reply:
one solution is sorting them upstream, with the help of tidy, and then place Ferrocene items below that list

other than that, I don't mind an occassional merge conflict... we deal with them all the time, and I don't even remember the last time we had an upstream pull without conflicts

Reply:
I think we should still minimize the amount of merge conflicts we get: the situation is already bad as it is, if we don't keep it under control it will keep getting worse.

I don't think sorting them upstream is a viable option either, because some of the ordering in that list is load bearing, as in some steps are expected to be executed after other steps when running with no explicit arguments.

I'd keep things as they are right now.

Reply:
> I'd keep things as they are right now.

should I revert the commit

Reply:
Yes.